tron/ssh.py

```diff
@@ -1,15 +1,23 @@
 import logging
 import struct
+import warnings
 
-from twisted.conch.client import default
+from cryptography.utils import CryptographyDeprecationWarning
 from twisted.conch.ssh import channel
 from twisted.conch.ssh import common
 from twisted.conch.ssh import connection
 from twisted.conch.ssh import keys
-from twisted.conch.ssh import transport
 from twisted.internet import defer
 from twisted.python import failure
 
+# We are ignoring CryptographyDeprecationWarning as it is coming from the twisted upstream library
+# The two deprecated warnings we are seeing are Blowfish and CAST5
+# Ideally we can remove this once removed in https://github.com/twisted/twisted/blob/trunk/src/twisted/conch/ssh/transport.py#L95-L109
```

Comment:
feel free to drop this suggestion - the current comment is good as well, i mostly just future-proofed it in my suggestion :p

```suggestion
# Ignore CryptographyDeprecationWarning as we don't use `cryptography` directly and
# the warnings are coming from one of our dependencies (Twisted) that does. There's
# nothing we can do until they stop using the deprecated ciphers - so ignoring these 
# warnings should be safe
```
(might need to mess with the line lengths if you want to switch to this comment